contracts/BoundaryManager.sol

```diff
@@ -91,8 +96,14 @@ contract BoundaryManager {
 
     function addRefundReceivers(address tokenAddress, address[] calldata refundReceivers) public {
         RefundBoundary storage safeRefundBoundary = safeRefundBoundaries[msg.sender][tokenAddress];
+        
+        // COMMENT: can this overflow the uint16 type and thus start the count from 0?
```

Comment:
Solidity 0.8 integrates SafeMath by default and it should revert if it overflows

Reply:
https://soliditydeveloper.com/solidity-0.8